0.download_data/metadata/scripts/generate_platemap_csvs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Generate platemap files from original position files
+# 
+# In this notebook, position files are used to generate plate map files to use for saving max projection images and annotating single-cell outputs downstream. 
+# Columns are updated to avoid any downstream issues.
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+
+
+# ## Generate platemap files
+
+# In[2]:
+
+
+# Dir path for output of platemap CSV files
+platemap_dir = pathlib.Path("./platemaps")
+platemap_dir.mkdir(parents=True, exist_ok=True)
+
+# Find all position txt files in the current directory starting with "slide"
+position_files = pathlib.Path().resolve().glob('slide*')
+
+# Instantiate a empty list to append updated plate maps to
+position_dfs = []
+
+# Iterate through each file to update "Point Name" and "Image" columns
+for file in position_files:
+    # Read the CSV file
+    df = pd.read_csv(file, delimiter='\t', encoding='utf-16')
```

Comment:
Is there a reason why utf-16 is used here instead of utf-8? If the data does not contain non-ASCII characters, consider using utf-8 for better memory efficiency. 

If there is a specific reason for using utf-16, please feel free to ignore this comment. 😊

Reply:
That was the encoding of the TXT file that was sent to us, so unfortunately we had to use it. Great point though!